src/test/java/com/powsybl/openloadflow/ac/LoadModelTest.java

```diff
@@ -62,8 +62,8 @@ void zipLoadModelTest() {
                 .add();
         LoadFlowResult result = loadFlowRunner.run(network, parameters);
         assertSame(LoadFlowResult.ComponentResult.Status.CONVERGED, result.getComponentResults().get(0).getStatus());
-        assertActivePowerEquals(48.765, zipLoad.getTerminal());
-        assertReactivePowerEquals(30.759, zipLoad.getTerminal());
+        assertActivePowerEquals(48.786, zipLoad.getTerminal());
```

Comment:
Why do we have these differences?

Reply:
there was a mistake in previous commit, a targetP instead of a targetQ